### ISSUE TITLE
Sort triangle hidden behind scroll bar

### ISSUE THREAD

**Issue by mherrmann** [fman-community/fman#307]
This was originally reported by @ThePlagueD0ctor in #301. It affects at least fman 0.8.7.
![image](https://user-images.githubusercontent.com/1076393/35904516-ab0bee9a-0be3-11e8-949a-f154825ac838.png)

@ThePlagueD0ctor, does the problem appear in all folders, or just `C:\Windows`? Does it also appear when you use `Ctrl+F3` to change the sort column?

**Comment by ThePlagueD0ctor** [comment]
Yes, the problems appear in all folders. If there is scrollbar - the triangle is not shown, if there is no scrollbar - the triangle is visible.
The problem can be reproduced with Ctrl+F3.
Maybe the reason is that i don't have SortSettings.json? But then - why i don't have it?)

**Comment by mherrmann** [comment]
Okay, thanks. It's strange that the triangle doesn't appear for you.

I believe it's not related to `Sort Settings.json`. See my latest comment / question in #301.
